packages/docs/src/css/colors.css

```diff
@@ -65,6 +65,7 @@
 
   --swm-red-dark-140: #5a3b46;
   --swm-red-dark-120: #914f55;
+  --swm-red-dark-110: #c86364;
```

Comment:
i don't think this color exist in the swm brandbook, you can add it somewhere else and named differently if necessary